feat(serverless): add serverless job retries MTA-6904 (#6602)
scaleway/docs-content: pages/serverless-jobs/how-to/manage-job-retries.mdx

---
title: How to manage the retries of a Serverless Job
description: Manage job retries efficiently in Scaleway Serverless.
tags: jobs serverless-jobs retries
dates:
  validation: 2026-06-15
  posted: 2026-06-15
---
import Requirements from '@macros/iam/requirements.mdx'

> The retry option allows you to define how many times a Serverless Job is automatically rerun in case of failure. When the maximum number of failed retries is reached, the job is permanently marked as failed.

Reviewer: ```suggestion
The retry option allows you to define how many times a Serverless Job automatically retries on failure. When the maximum number of failed retries is reached, the job is permanently marked as failed.
```

> The retry option allows you to define how many times a Serverless Job is automatically rerun in case of failure. When the maximum number of failed retries is reached, the job is permanently marked as failed.

Reviewer: You use "retry" and "rerun". Prefer the word used in the console and use it consistently.

Author: The retry triggers the rerun. The workflow is run > failure > retry > rerun (or back to run). The user can set retries. The reruns are the consequences of it. We cannot set reruns.

<Requirements />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- [Created a Serverless Job](/serverless-jobs/how-to/create-job/)

1. Click **Jobs** in the **Serverless** section of the side menu. The jobs page displays.

2. Click the name of the job you want to manage, then click the **Settings** tab.

3. Scroll down to the **Configure advanced options** section, then click the **Retries** tab.

> 4. Set the number of times you want your job to be rerun in case of failure. The default value is 0, and the maximum value is 5.

Reviewer: ```suggestion
4. Set the number of times the job retries on failure. The default value is 0, and the maximum value is 5.
```
